components/Profile/ParentProfile/StudentClasses.tsx

```diff
@@ -0,0 +1,110 @@
+import { Class } from "../../../models/";
+import { APIContext } from "../../../context/APIContext";
+import styles from "./StudentClasses.module.css";
+import React, { useContext, useEffect, useState } from "react";
+import RRule from "rrule";
+import {
+  Accordion,
+  AccordionItem,
+  AccordionItemHeading,
+  AccordionItemButton,
+  AccordionItemPanel,
+} from "react-accessible-accordion";
+import AccessTime from "@mui/icons-material/AccessTime";
+import CalendarMonth from "@mui/icons-material/CalendarMonth";
+import School from "@mui/icons-material/School";
+
+type StudentClassesProps = {
+  id: string;
+};
+
+const getFormattedTime: string = (time: string) => {
+  const hours24 = parseInt(time.substring(0, 2));
+  const hours = ((hours24 + 11) % 12) + 1;
+  const hoursText = hours < 10 ? "0" + hours.toString() : hours.toString();
+  const amPm = hours24 > 11 ? " pm" : " am";
+  const minutes = time.substring(3, 5);
+
+  return hoursText + ":" + minutes + amPm;
+};
```

Comment:
just wondering, is there no better way to do this? this forces us to keep the exact same formatting through the entire app. Maybe luxon would help here? B/c now we have to make sure the formatting is correct

Reply:
Yes, I agree I think something like luxon is a cleaner option. Added in new iteration